### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by cfarrell1980**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -425,7 +425,7 @@ <h5 lang="en">Change language</h5>
         </div>
         <div class="clearfix"></div>
       </div>
-      <p class="text-center">© 2015 SUSE LLC. <span lang="en">All Rights Reserved.</span></p>
+      <p class="text-center">© 2015 SUSE LLC. <span lang="en">All Rights Reserved. </span><span lang="de"><a href="https://de.opensuse.org/Impressum">Impressum</a>.</span></p>
```
Comment (lnussel) [inline index.html]: shouldn't this point to en.opensuse.org?

Comment (cfarrell1980) [inline index.html]: it is only really relevant for Germany. It certainly could point to en.opensuse.org - and I even created en.opensuse.org/Impressum for it. I didn't know what the <span lang="en"> did though - does it mean visitors with e.g. german browsers won't see that?

Comment (cfarrell1980) [inline index.html]: Github just removed the html - I meant to say I don't know what span lang=en does.

Comment (lnussel) [inline index.html]: the lang attribute marks the string or translation. So in the current form translators would translate both "Impressum" as well as where the link points at. What's correct depends on what is needed really. Do you want the "Impressum" to be only displayed when the language is set to German, or do you want to also have an "Imprint" link that is always shown in any language?

Comment (cfarrell1980) [inline index.html]: As long as german visitors see "Impressum", I'm not too pushed. Imprint for english visitors would work, of course (but they would end up going to de.opensuse.org/Impressum, no?

Comment (lnussel) [inline index.html]: no necessarily. as long as the link is part of the translated string the target can be changed per language

**Pull request merged**